config/checkstyle/checkstyle.xml

```diff
@@ -326,7 +326,7 @@
                       value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
         </module>
         <module name="JavadocMethod">
-            <property name="scope" value="public"/>
+            <property name="accessModifiers" value="public"/>
```

Comment:
this modification is not related to the current pr

Reply:
Fixed.